src/MSBuild/Microsoft.Build.xsd

```diff
@@ -1,11 +1,11 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
 <xs:schema targetNamespace="http://schemas.microsoft.com/developer/msbuild/2003" 
 xmlns:xs="http://www.w3.org/2001/XMLSchema"
 xmlns:msb="http://schemas.microsoft.com/developer/msbuild/2003"
 elementFormDefault="qualified">
 
     <!-- =================== IMPORT COMMON SCHEMA =========================== -->
-    <xs:include schemaLocation="MSBuild\Microsoft.Build.CommonTypes.xsd"/>
+    <xs:include schemaLocation="MSBuild/Microsoft.Build.CommonTypes.xsd"/>
```

Comment:
Can you revert the changes in this file back to the contents of `master` please?

Reply:
Done